api/src/components/publicationVersion/__tests__/getAll.test.ts

```diff
@@ -52,7 +52,7 @@ describe('Get many publication versions', () => {
         });
 
         expect(getPublications.status).toEqual(200);
-        const publicationDates = getPublications.body.data.map((version) => version.publishedDate);
+        const publicationDates: Date[] = getPublications.body.data.map((version) => version.publishedDate as Date);
```

Comment:
Can typescript infer the type of this from the return type of the map?

Reply:
I would have thought so but if I remove it and hover over it, it's typed as any.